bin/fixedcase/special-case-titles

```diff
@@ -0,0 +1,98 @@
+ECONOMY IS A PERSON: a Chinese-English corpora and ontological-based comparison using the Conceptual Mapping Model
```

Comment:
I didn't look at the paper, but are the first four words really always all caps?

Reply:
That's part of the standard notation when naming a metaphor.